conf/nd96amsr_v4.conf

```diff
@@ -24,37 +24,37 @@
  * || check_fs_mount_rw -t "devtmpfs" -s "devtmpfs" -f "/dev"
  * || check_fs_mount_rw -t "devpts" -s "devpts" -f "/dev/pts"
  * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run"
- * || check_fs_mount_rw -t "ext4" -s "/dev/root" -f "/"
```

Comment:
Are the tests being deleted not needed?

Reply:
in short yes. They don't add much value and are not uniform across all OS's